discovery-provider/alembic/trigger_sql/ddl.sql

```diff
@@ -252,3 +257,40 @@ BEGIN;
 
   delete from users where txhash='0x7bc1e2c100a04061098db053957072b7eb0db75ab8f5c35873b56f6b6b817d9e'; -- tx without handle from another wallet
 COMMIT;
+
+-- 5/31/23 Rename delegations -> grants, app_delegates -> developer_apps. Remove shared_address from delegations.
```

Comment:
maybe easier to rename the delegations table above and then drop the delegation table here? either way is good tho